Fixing line wrapping in CONTRIBUTING.rst

JackDanger opened the pull request:
As a new contributor (who doesn't know about Tox) it was hard to figure out what I needed to `pip install` in order to get things going. Rather than making every developer look inside `.travis.yml` for the install command this PR uses the standard requirements.txt file for pip.

Tarrasch commented:
I really like the line wrapping commit. And I like the splitting of the commits even more. :)

But I think a `requirements.txt` in the project root would be confusing. That's not the requirements for luigi, it's the requirements for running the tests only. Can you instead somehow make the "Running Unit Tests" refer to CONTRIBUTING.rst? Or somehow merge them? Does that make sense? :)

JackDanger commented:
@Tarrasch I've removed the second commit (the one that added the requirements.txt) and renamed this to be about fixing the line wrapping. I'll make a second PR that tries to address the root problem here. Thank you!

Tarrasch commented:
Awesomeness!

Tarrasch commented:
Also, when I say "merge", I would like to at least keep the file CONTRIBUTING.rst, as github integrates with it when contributors sends PRs.

JackDanger commented:
@Tarrasch I tried to address this in #1126. Let me know what you think!
